src/content/docs/ecosystem/tachyonfx.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Shader animations (Tachyonfx)
```

Comment:
I think "Effects and animations" sound better than "Shader animations". It doesn't mislead anyone to thinking it involves a GPU + shaders sound complicated/requiring a different skill set.

Reply:
Thanks for the review, I went with "Effects and animations :)